assets/src/application/ui/calendars/dateDisplay/CalendarDateSwitcher/index.tsx

```diff
@@ -12,8 +12,10 @@ const CalendarDateSwitcher: React.FC<CalendarDateSwitcherProps> = React.memo(
 	({ className, displayDate = DisplayStartOrEndDate.start, labels, ...props }) => {
 		const startDate = parseISO(props.startDate) || PLUS_ONE_MONTH;
 		const endDate = parseISO(props.endDate) || PLUS_TWO_MONTHS;
+		const { footerPast, footerFuture, footer = '' } = labels;
+		const footerText = footerPast && footerFuture ? switchTenseForDate(endDate, footerPast, footerFuture) : footer;
+
 		const headerText = switchTenseForDate(startDate, labels?.headerPast ?? '', labels?.headerFuture ?? '');
```

Comment:
can you apply the same change you made for the footer text to the header text as well plz?

Reply:
thx, done